source/envs/env2048.py

```
@@ -0,0 +1,540 @@
+    def step(self, action: int):
+        if self.done:
+            print(f"max value : {self.board.max()}")
+            return (
+                self._get_obs(),
+                0.0,
+                True,
+                False,
+                {"score": self.score, "won": self._won()},
+            )
+
+        assert self.action_space.contains(action), "Invalid action"
```

Ruff (0.12.2)

Use of `assert` detected

(S101)